rasa/core/processor.py

```diff
@@ -324,22 +322,53 @@ def _has_message_after_reminder(
             or not self._is_reminder_still_valid(tracker, reminder_event)
         ):
             logger.debug(
-                f"Canceled reminder because it is outdated. "
-                f"(event: {reminder_event.action_name} id: {reminder_event.name})"
+                f"Canceled reminder because it is outdated. " f"({str(reminder_event)})"
```

Comment:
`f"` already takes care of that
```suggestion
                f"Canceled reminder because it is outdated. " f"({reminder_event})"
```